src/python/tritonfrontend/CMakeLists.txt

```diff
@@ -96,17 +94,76 @@ endif()
 
 if(${TRITON_ENABLE_TRACING})
   message("TRACING/STATS IS CURRENTLY NOT SUPPORTED.")
-  find_package(absl CONFIG REQUIRED)
-  find_package(CURL CONFIG REQUIRED)
-  find_package(nlohmann_json CONFIG REQUIRED)
-  find_package(opentelemetry-cpp CONFIG REQUIRED)
-  list(APPEND PY_BINDING_DEPENDENCY_LIBS
-    tracing-library
+  list(
+      APPEND PY_BINDING_DEPENDENCY_LIBS
+      tracing-library
   )
 endif()
 
 # ===================== End of Collection ===================================
 
+# ========== Create support-lib (Static Lib for Dependencies) ================
+add_library(
+  support-lib
+    STATIC
```

Comment:
Just curious - What's the file size difference of the wheel when using STATIC vs before?

Reply:
Wheel size was roughly 10 MB before using STATIC and after as well.